IfcDiscreteAccessory Rail Import BIMQ 2021-07-21T07:41:33.5120880Z (#187)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DiscreteAccessoryTypeInsulator/Documentation.md

> Properties of insulator used in railway. The property set can be used by the predefined type INSULATOR of IfcDiscreteAccessory

Reviewer: ```suggestion
Properties of insulator used in railway. The property set can be used by the predefined type INSULATOR of IfcDiscreteAccessory.
```

> Properties of insulator used in railway. The property set can be used by the predefined type INSULATOR of IfcDiscreteAccessory.

Reviewer 2: ```suggestion
Properties of an insulator. The property set can be used by the predefined type INSULATOR of IfcDiscreteAccessory.
```
